src/styles/global.css

```diff
@@ -0,0 +1,31 @@
+/*Navbar Component*/
+
+#Navbar {
+    width: 100%;
+    background-color: #333333;
+}
+
+#LinkList {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    list-style: none;
+    padding: 0px;
+    margin: 0px;
+}
+
+.IndividualLinks {
+    display: block;
+    text-decoration: none;
+    color: white;
+    padding: 14px 16px;
+}
```

Comment:
Nothing wrong with this, but worth looking into encapsulating styles within a component instead of jamming them in globally.